src/Elm/Parser/Layout.elm

```diff
@@ -18,37 +18,58 @@ import Rope exposing (Rope)
 import Set
 
 
-nonEmptyWhiteSpaceOrComment : Parser.Parser (Maybe (Node String))
-nonEmptyWhiteSpaceOrComment =
+{-| Nothing if there can't be more whitespace/comments ahead,
+otherwise Just with the next comment node
```

Comment:
This documentation comment is out of date in the last state of the PR

Reply:
thanks, fixed.